artifact/utils.go

```diff
@@ -25,6 +26,20 @@ type TestDirEntry struct {
 	IsDir   bool
 }
 
+func MakeFakeUpdate(data string) (string, error) {
+	f, err := ioutil.TempFile("", "test_update")
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	if len(data) > 0 {
+		if _, err := f.WriteString(data); err != nil {
+			return "", err
+		}
+	}
+	return f.Name(), nil
+}
+
```

Comment:
The issue is that you can not share test files from one package with other package that's why this needs to stay here. I see the point, so I will move this to tests itself, which unfortunately will cause some code duplication.